arm-sql/2015-05-01-preview/swagger/advisors.json

```diff
@@ -0,0 +1,981 @@
+          }
+        },
+        "x-ms-examples": {
+          "Update server advisor": {
+            "$ref": "../examples/ServerAdvisorUpdate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {}
+    },
+    "AdvisorProperties": {
+      "description": "Properties for a Database, Server or Elastic Pool Advisor.",
+      "required": [
+        "autoExecuteStatus"
+      ],
+      "type": "object",
+      "properties": {
+        "advisorStatus": {
+          "description": "Gets the status of availability of this advisor to customers. Possible values are 'GA', 'PublicPreview', 'LimitedPublicPreview' and 'PrivatePreview'.",
+          "enum": [
+            "GA",
+            "PublicPreview",
+            "LimitedPublicPreview",
+            "PrivatePreview"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AdvisorStatus",
+            "modelAsString": true
+          }
+        },
+        "autoExecuteStatus": {
+          "description": "Gets the auto-execute status (whether to let the system execute the recommendations) of this advisor. Possible values are 'Enabled' and 'Disabled'",
+          "enum": [
+            "Enabled",
+            "Disabled",
+            "Default"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutoExecuteStatus",
+            "modelAsString": true
```

Comment:
Are you likely to add new elements to this enum in the foreseeable future? If no, use modelAsString=false instead.